syntax = "proto3";

import "svg.proto";

package pr_map_renderer;

message RenderSettings {
    double width = 1;
    double height = 2;
    double padding = 3;
    double line_width = 4;
    double stop_radius = 5;
    uint32 bus_label_font_size = 6;
    pr_svg.Point bus_label_offset = 7;
    uint32 stop_label_font_size = 8;
    pr_svg.Point stop_label_offset = 9;
    pr_svg.Color underlayer_color = 10;
    double underlayer_width = 11;
    repeated pr_svg.Color color_palette = 12;
}
